MobiusCore/Source/EffectHandlers/EffectRouterDSL.swift

```diff
@@ -61,14 +69,12 @@ public extension PartialEffectRouter {
         _ connectable: C
     ) -> EffectRouter<Input, Output> where C.InputType == Payload, C.OutputType == Output {
         var connection: Connection<Payload>?
-        return to(EffectHandler<Payload, Output>(
-            handle: { payload, output in
-                connection = connection ?? connectable.connect(output)
-                connection?.accept(payload)
-            },
-            disposable: AnonymousDisposable {
+        return to { payload, response in
```

Comment:
This will leak memory since `response.end()` is never called, so `EffectHandlingConnection` will accumulate entries for each effect it receives.

Reply:
I think the `Response` design conflicts with [backwards compatibility for connectables](https://github.com/spotify/Mobius.swift/pull/95), so this function should be removed.

The alternative design of having one dispose/end per handler rather than per invocation maps cleanly to connectables.